estate/models/estate_property.py

```diff
@@ -1,4 +1,4 @@
-from odoo import fields, models
+from odoo import fields, models, api
```

Comment:
Usually we try to order it alphabetically
```suggestion
from odoo import api, fields, models
```